gems/parallel/1.20/parallel.rbs

```diff
@@ -0,0 +1,83 @@
+module Parallel
+  class Break < StandardError
+    attr_reader value: untyped
+
+    def initialize: (?untyped value) -> void
+  end
+
+  class Kill < Break
+  end
+
+  Stop: Object
+
+  VERSION: String
+
+  Version: String
+
+  def self.all?: [T] (Enumerable[T] | (^() -> T | untyped) array, # TODO: `T | Stop` should be better than `T | untyped`.
```

Comment:
[question] When using `Stop` instead of `untyped`, 

```diff
--- a/gems/parallel/1.20/parallel.rbs
+++ b/gems/parallel/1.20/parallel.rbs
@@ -14,7 +14,7 @@ module Parallel
 
   Version: String
 
-  def self.all?: [T] (Enumerable[T] | (^() -> T | untyped) array, # TODO: `T | Stop` should be better than `T | untyped`.
+  def self.all?: [T] (Enumerable[T] | (^() -> T | Stop) array, # TODO: `T | Stop` should be better than `T | untyped`.
                       ?in_processes: Integer,
                       ?in_threads: Integer,
                       ?progress: _ToStr,
```

then the following error occurs. How can I fix it? 🤔 

```console
$ ../../../bin/run _scripts/test --repo=../../..
...
gems/parallel/1.20/parallel.rbs:17:50...17:54: Could not find ::Stop (RBS::NoTypeFoundError)
...
```

Reply:
RBS doesn't treat constants as types. So you cannot access `Stop` as a type in RBS.
I guess it will be useful if we can put a constant as a type, so could you open an issue to ruby/rbs repo? cc/ @soutaro 

By the way, we can improve the type by using a literal type instead.


```rbs
class Parallel
  type stop = :__type_of_parallel_stop__
  Stop: stop
end
```

it solves the problem, but it is incorrect. So personally I think untyped is better.

Reply:
@pocke Thanks for your kind feedback! I've opened an issue: ruby/rbs#623. 👍 

Could you please provide further information if necessary?